automated install/basic-install.sh

```diff
@@ -1048,6 +1051,12 @@ main() {
 	echo ":::"
 	echo "::: The install log is located at: /etc/pihole/install.log"
 	echo "::: View the web interface at http://pi.hole/admin or http://${IPv4_address%/*}/admin"
+
+    [ ${INSTALLBUG} ] && (
+        echo "*** ***************************************************************************"
+        echo "*** INSTALLER ENCOUNTERED ONE OR MORE BUGS, PLEASE REVIEW LOGS AND CHECK INSTALLATION"
+        read -p "*** PLEASE PRESS ENTER TO CONFIRM "
```

Comment:
If the script is `curl`'d  The script will fall out here anyway. Found that out a while back! Curling to bash doesn't support reading user input.